src/pages/Signup/Signup.jsx

```diff
@@ -25,6 +46,27 @@ export const Signup = () => {
 
 	const handleFormSubmit = async (event) => {
 		event.preventDefault();
+		const isValidForm = validateSignupForm(errorDispatch, credentials);
+		if (isValidForm) {
+			try {
+				const {
+					data: { createdUser, encodedToken },
+				} = await axios.post("/api/auth/signup", credentials);
+
+				updateUser(createdUser);
+				cartDispatch({ type: UPDATE_USER_CART, payload: createdUser.cart });
+				cartDispatch({
+					type: UPDATE_USER_WISHLIST,
+					payload: createdUser.wishlist,
+				});
+
+				localStorage.setItem("token", encodedToken);
+				errorDispatch({ type: CLEAR_SIGNUP_FORM });
+				navigate("/");
+			} catch (error) {
+				console.log("Something is wrong, please try later.");
+			}

```

Comment:
error shows up in console,show it on the ui instead

Reply:
I will change that.